### ISSUE TITLE
Guilting and Manipulative behaviors

### ISSUE THREAD

**Issue by chrisgraham** [ContributorCovenant/contributor_covenant#793]
A common problem in Open Source communities is people trying to push people into things, by making demands and/or making people feel guilty. Related, trying to manipulate people into doing what you want. This can cause a lot of psychological harm.

For example, demanding a maintainer implements a new feature.
For example, a maintainer trying to manipulate someone into being a contributor by hanging carrots that probably will never actually exist.

I think it would be good to cover this with a couple of new items on the unacceptable behavior list:

> Making demands on volunteers to take on responsibilities they have not agreed to, or otherwise making people feel guilty for not meeting your particular needs
> Trying to manipulate people into doing what you want against their own interests, particularly through deceptive behavior

**Comment by fccm** [comment]
I agree that this is something that often occur in the open source community.

I even applied to a job to an open source company, and they officially requested me to become an active contributor to php-symphony with code accepted upstream to consider my candidacy for the job.
